.black8

```diff
@@ -0,0 +1 @@
+black8
```

Comment:
🐶 🍝

Reply:
not the best name :p

especially given that this bundles our formatting stuff... let's namespace this somehow to us?

Reply:
We definitely can't start a package name with a digit though. I guess I am fine with fourmat (*especially* because it's in the tradition of naming 4C packages starting with f)

Reply:
iblort8-4catalyzer?